### PR TITLE
Add "why" for branding and licencing

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

**Comment by potiuk** [comment]
As the next "why" I proposed some "why" for branding/licencing in the "advisors" working group. This is my understanding on why licensing and branding and taking care about it is the most important thing from the legal point of view for all PMC members, but I woudl love to hear what others think about it.

File: wg-advisors/why/why_branding_licencing.md [hunk 1]
```diff
@@ -1,13 +1,68 @@
...
 ## Summary
 
+The Apache Software Foundation's most important asset is a combination of branding and license that is
+used by all Apache projects. The license is the Apache License, Version 2.0, and the branding is based
+on the project name. The licence is a permissive open source license that allows the software to be used
+by anyone for any purpose, and the branding is a set of rules that ensure that the software is recognized
+as an Apache project and that the project is recognized as part of the Apache Software Foundation.
+
+The software that ASF projects produce is released under the Apache License, Version 2.0, which is a permissive
+open source license that allows the software to be used by anyone for any purpose and it is important that
```
Comment (rbowen) [inline wg-advisors/why/why_branding_licencing.md:12]: This paragraph duplicates much of what is said in the first paragraph. Can we consolidate a little here?

File: wg-advisors/why/why_branding_licencing.md [hunk 2]
```diff
@@ -1,13 +1,68 @@
...
+we do not limit anyone from using the software in any way. And while we allow anyone to use the software,
+and redistribute it, we do not allow anyone to claim that they are the "owners" of the software, or that
+they are controlling the project.
+
+This is why it is important to ensure that the project's brand is not used in a way that could cause confusion
+or mislead users on who is owning or controlling the project. This includes ensuring that the project's name
+is not used in a way that could imply that another party is controlling the project - assuring
+vendor neutrality.
```
Comment (rbowen) [inline wg-advisors/why/why_branding_licencing.md:20]: Perhaps a link here to one of the docs where we discuss vendor neutrality in greater detail.

File: wg-advisors/why/why_branding_licencing.md [hunk 3]
```diff
@@ -1,13 +1,68 @@
...
 
+* [The Apache Software Foundation Trademark Policy](https://www.apache.org/foundation/marks/)
+* [The Apache Licences](https://apache.org/licenses/)
+* [Third party event branding policy](https://www.apache.org/foundation/marks/events.html)
+
 ## Why are we doing it?
 
+The software that ASF projects produce is released under the Apache License, Version 2.0, which is a permissive
+open source license that allows the software to be used by anyone for any purpose and it is important that
+we do not limit anyone
```
Comment (rbowen) [inline wg-advisors/why/why_branding_licencing.md:51]: This, again, repeats the first paragraph.

File: wg-advisors/why/why_branding_licencing.md (hunk 1 above)
Comment (potiuk) [reply wg-advisors/why/why_branding_licencing.md:12]: Yes. Merged the two. From the second paragraph I left only the "owning" part.

File: wg-advisors/why/why_branding_licencing.md (hunk 2 above)
Comment (potiuk) [reply wg-advisors/why/why_branding_licencing.md:20]: Good point. Added links to [Project Independence](https://community.apache.org/projectIndependence.html)

File: wg-advisors/why/why_branding_licencing.md (hunk 3 above)
Comment (potiuk) [reply wg-advisors/why/why_branding_licencing.md:51]: Right. Removed it.

**Comment by potiuk** [comment]
Applied comments.

**Pull request merged**